packages/core/useFileDialog/index.ts

```diff
@@ -0,0 +1,52 @@
+import { readonly, ref } from 'vue-demi'
+
+export interface UseFileDialogOptions {
```

Comment:
I suggest adding a new option `capture`, to select the file input source.

Reply:
Can elaborate?